terms.html

```diff
@@ -3,14 +3,23 @@
 </p>
 
 <dl class="termlist">
+  <dt><dfn data-lt="author|authors|authored|author's|author(s)|authoring">author</dfn></dt>
+  <dd>
+A role that a <a>party</a> performs as it expresses an assertion about a particular <a>entity</a>, i.e., creates a <a>claim</a>.
+  </dd>
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
+A digital representation of an assertion made about an <a>entity</a> by a specific <a>party</a>.
```

Comment:
why must it specifically be a digital representation?

Reply:
While it may not be necessary, I think it might be helpful. I would argue as follows:

1. we define terms, the scope of which is this standard. This means that however we define a term here, that doesn't mean it must have that meaning in other contexts. See [my comment on #1047](https://github.com/w3c/vc-data-model/issues/1047#issuecomment-1437973501) for an elaboration on this.
2. within this scope, we should define terms as we need them. AFAICT, a reader of the standard should not consider 'claim' to mean, e.g., someone saying something about an entity (a sound), or a physical document that has texts, or pictures, about an entity. It would need to be digital.



as it prevents people from considering that a claim might also refer to, e.g., something said by someto think of claims as being, e.g. someone shouting one, or a physical (paper) document, or something else they may think of.